src/common/layouts/sidebar/variants/Courses.tsx

```diff
@@ -136,7 +131,7 @@ export function Courses({ term }: Props): JSX.Element | null {
 
   return (
     <>
-      <CoursesTopBar onFilter={handleFilter} />
+      <CoursesTopBar onFilter={handleFilter} filter/>
```

Comment:
I'm curious, does this work as expected? Since `filter` is a boolean in the props type, my understanding is naming the prop as you've done here will mean it's always true. Doing `filter={filter}` would correctly pass the variable that can be true or false.
I'm not 100% sure I'm right though so if this works just let me know